### ISSUE TITLE
[$10][MSFT-60] [Register] "By clicking "Join" you agree to Topcoder's Terms and Privacy Policy" text foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2776]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Check the "By clicking "Join" you agree to Topcoder's Terms and Privacy Policy" text

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #3A3A3A. New contrast ratio: 4.55
- Passes AA, if you change text color to #757585. New contrast ratio: 4.53

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**
Element has insufficient color contrast of 2.49 (foreground color: #a3a3ae, background color: #ffffff, font size: 9.0pt, font weight: normal).

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F33

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096293 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096293 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096293 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/appirio-tech/accounts-app/pull/227

**Comment by tosha5252** [comment]
Color of text set at #A3A3AE, which does not pass WCAG standards.

Env:

Windows 10
Browser: Chrome

**Comment by mishacucicea** [comment]
@tosha5252 - on what environment did you tested this? I can see in the code change that the color is #A3A3AE ?

**Comment by PrakashDurlabhji** [comment]
@tosha5252 color is changed as mentioned in issue description. is there anything missed?
@mishacucicea

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096968 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096968 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR for this is proper. dont know why QA fail label is still present.
PR for this was https://github.com/appirio-tech/accounts-app/pull/227

**Comment by mishacucicea** [comment]
@tosha5252 Can you please retest this one?

**Comment by drasticdpk** [comment]
@mishacucicea still having issue, I am not sure whether it is deployed or not. @lakshmiathreya FYI
![image](https://user-images.githubusercontent.com/25297859/61953689-c5542900-afd6-11e9-9085-86962f456a8d.png)

**Comment by drasticdpk** [comment]
@PrakashDurlabhji 
 We can still see the old color code , here is the reference. 
![image](https://user-images.githubusercontent.com/25297859/61996883-77fcb800-b0b9-11e9-90ed-5f17ecc1b34e.png)
I am not sure this code has merged successfully or not FYI @veshu  @mishacucicea @tosha5252 @lakshmiathreya 
https://github.com/appirio-tech/accounts-app/pull/227/files

**Comment by tosha5252** [comment]
I am still seeing #a3a3ae

**Comment by PrakashDurlabhji** [comment]
@tosha5252 code is merged but i guess deployment issues.
@drasticdpk

**Comment by mishacucicea** [comment]
@nkumar-topcoder @sushilshinde Is this a deployment issue?

**Comment by nkumar-topcoder** [comment]
But code is correct pointing to color: #3A3A3A; could be cache issue, I have refreshed the cache now can you please check and confirm @mishacucicea @drasticdpk @tosha5252 @PrakashDurlabhji

**Comment by mishacucicea** [comment]
Issue looks OK on dev as well.
![image](https://user-images.githubusercontent.com/1721199/62006157-b770e600-b145-11e9-98bd-d554960e0810.png)

**Comment by tosha5252** [comment]
![Annotation 2019-07-28 105444](https://user-images.githubusercontent.com/6400904/62009572-81703980-b126-11e9-8734-7dc4b51e98d5.png)
![Annotation 2019-07-28 105621](https://user-images.githubusercontent.com/6400904/62009573-81703980-b126-11e9-83f3-1180bddd2f85.png)

Still seeing the same thing....checked Firefox, Chrome, Edge and using WAVE tool

**Comment by PrakashDurlabhji** [comment]
@tosha5252 @mishacucicea is my PR correct then? or still deployment issue?

**Comment by mishacucicea** [comment]
@PrakashDurlabhji @tosha5252 @drasticdpk  It's very odd, the code is there, it works when I access it (as seen in the screenshot). 

Have you tried a hard refresh or incognito window?

**Comment by nkumar-topcoder** [comment]
> @PrakashDurlabhji @tosha5252 @drasticdpk It's very odd, the code is there, it works when I access it (as seen in the screenshot).
> 
> Have you tried a hard refresh or incognito window?
@tosha5252 can you please check.

**Comment by PrakashDurlabhji** [comment]
@tosha5252 @mishacucicea such similar deployment issue is found even on nahids PR, though his PR merged but still no code on environment, I guess this kind of problem is on similar line. am I correct?

**Comment by mishacucicea** [comment]
I can see your change on the environment, so from my side, it's good. But testers need to confirm as well, in case we miss other details.

**Comment by ajefts** [comment]
I tested this in dev and prod and I'm not able to see any change:

![image](https://user-images.githubusercontent.com/3812270/62093776-87f6d200-b248-11e9-9637-5d8ea6b912dd.png)

@nkumar-topcoder Maybe we need to check the deploy or cloudfront cache?

**Comment by tosha5252** [comment]
@mishacucicea checked in both dev and prod, in both Chrome incognito and Windows 10 Edge InPrivate and still showing the same thing...a3a3ae

**Comment by nkumar-topcoder** [comment]
@tosha5252 re-check in topcoder-dev.com. I just verified the code its actually  #3A3A3A; I have refreshed CDN.

**Comment by nithyaasworld** [comment]
@nkumar-topcoder This one is still failing too.

![image](https://user-images.githubusercontent.com/5114190/62129659-eaa5a780-b2f4-11e9-9020-e8920fe31b21.png)

**Comment by PrakashDurlabhji** [comment]
@sushilshinde I hope I can wait for discussion above, I am still confused whether deployment issue exist? because locally all working.

also, I guess rohit has been given special privilges to unassign me and start assigning it, though discussion is going on ticket and not a feedback pending since days.

**Comment by lakshmiathreya** [comment]
Continue to see a3a3ae ...

**Comment by nithyaasworld** [comment]
@lakshmiathreya  Yeah, but based on our Slack conversations, made this to "QA Pass"

**Comment by ajefts** [comment]
Verified in prod.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096968<br/><br/>```This is an automated message for crazyk via Topcoder X```
